db/migrate/20230512095053_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Please note that in your migration folder, you have multiple files with the name `create_comments`. Kindly see to this and make sure its fixed 👍🏽
![Screenshot from 2023-05-19 10-16-18](https://github.com/habasm/BlogApp/assets/44624138/1c2d8d96-4ae3-4650-8839-6679a2964dfc)